src/components/crowdsale/utils.js

```diff
@@ -274,14 +274,65 @@ let fillCrowdsalePageStoreDates = (startsAtMilliseconds, endsAtMilliseconds) =>
   logger.log('endDate:', endsAtMilliseconds)
 }
 
+/**
+ * Check if a crowdsale is finalized
+ * @param methods
+ * @param crowdsaleExecID
+ * @returns {Promise<*>}
+ */
 export const isFinalized = async ({ methods }, crowdsaleExecID) => {
   const { addr } = contractStore.abstractStorage
   let params = []
   if (crowdsaleExecID) {
     params.push(addr, crowdsaleExecID)
   }
-  const { is_finalized } = await methods.getCrowdsaleInfo(...params).call()
-  return is_finalized
+  const crowdsale = await methods.getCrowdsaleInfo(...params).call()
+  const { isMintedCappedCrowdsale, isDutchAuction } = crowdsaleStore
+  let isFinalized = false
+  if (isMintedCappedCrowdsale) {
+    //Value is finalized is the index #3
+    isFinalized = crowdsale[3]
```

Comment:
Would you mind to keep track of the name of the property, if the returned item is an object:
`isFinalized = crowdsale.is_finalized || crowdsale[3]` ?

It will help us to be aware of the name of the property in smart-contracts

Reply:
Done!